test/fixtures/render/content-type.apib

```diff
@@ -0,0 +1,14 @@
+# GET /
++ Response 200 (application/json; charset=utf-8)
+    + Attributes
+        + user: pksunkara
+
+# GET /hal
++ Response 200 (application/hal+json)
+    + Attributes
+        + user: pksunkara
+
+# GET /typo
++ Response 200 (application/haljson)
```

Comment:
Ok. Can you please fix the regex so it works with `application/json` or `+json` correctly?